admin_app/src/app/content/edit/page.tsx

```diff
@@ -259,5 +320,62 @@ const Header = ({ content_id }: { content_id: number | null }) => {
     </Layout.FlexBox>
   );
 };
+const SavedContentDialog = ({
```

Comment:
This is the modal to either redirect to the main page or to the edit page to add/edit another content